dashboard/app/views/levels/_multi.html.haml

```diff
@@ -3,9 +3,15 @@
   - data = @level.properties
   - app = 'multi'
 
-  = render partial: 'levels/content', locals: {app: app, data: data, content_class: 'question'}
+  - question_content_blank = data['content1'].blank? && data['content2'].blank? && data['content3'].blank? && data['markdown'].blank?
+  = render partial: 'levels/content', locals: {app: app, data: data, content_class: question_content_blank ? nil : 'question'}
 
-  .answers
+  .answers{style: question_content_blank ? 'width: 100%' : ''}
+    - if data['options'].try(:[], 'answer_line')
```

Comment:
Same comment here about always having `data['options']` so you don't have to `.try()`.